content/developer/reference/frontend/patching_code.rst

```diff
@@ -53,57 +47,41 @@ Here is a simple example of how an object can be patched:
     },
   };
 
-  patch(object, "patch name", {
+  patch(object, {
     fn() {
       // do things
     },
   });
 
 
 When patching functions, we usually want to be able to access the ``parent``
-function.  Since we are working with patch objects, not ES6 classes, we cannot
-use the native ``super`` keyword. So, Odoo provides a special method to simulate
-this behaviour: ``this._super``:
+function.  To do so, we can simply use the native ``super`` keyword:
 
 .. code-block:: javascript
 
-  patch(object, "_super patch", {
+  patch(object, {
     fn() {
-      this._super(...arguments);
+      super.fn(...arguments);
       // do other things
     },
   });
 
 .. warning::
 
-  ``this._super`` is reassigned after each patched function is called.
-  This means that if you use an asynchronous function in the patch then you
-  cannot call ``this._super`` after an ``await``, because it may or may not be
-  the function that you expect.  The correct way to do that is to keep a reference
-  to the initial ``_super`` method:
-
-  .. code-block:: javascript
-
-    patch(object, "async _super patch", {
-      async myAsyncFn() {
-        const _super = this._super.bind(this);
-        await Promise.resolve();
-        await _super(...arguments);
-        // await this._super(...arguments); // this._super is undefined.
-      },
-    });
-
+    ``super`` can only be used in a method not a function. This means that syntaxes
```

Comment:
`syntaxes` is a little weird. Maybe "the following constructs". Also maybe make this into a code block instead of inline code if possible, it will be easier to read, and you can add inline comments with the error that would be raised